internal/config/config.go

```diff
@@ -407,6 +407,14 @@ func New(fileHandler file.Handler, name string, force bool) (*Config, error) {
 		c.MicroserviceVersion = Default().MicroserviceVersion
 	}
 
+	// TODO(3u13r): Remove this deprecation warning and enforce assigned managed identity after the v2.8.0 but before the v2.9.0 release.
+	if c.Provider.Azure != nil &&
+		(c.Provider.Azure.AppClientID != "" || c.Provider.Azure.ClientSecretValue != "") {
```

Comment:
We already have the validation function [`warnDeprecated`](https://github.com/edgelesssys/constellation/blob/757fccf8260b34afb5008c015600e32a6c16458a/internal/config/validation.go#L518).
The fields should either use the same function to print a warning, or we should align the style of the warning message.